packages/core/src/definitions/sdkFeatures.ts

```diff
@@ -0,0 +1 @@
+export const features = {};
```

Comment:
I made it more generic and unopiniated for now so right now it's typed as `Record<string | unknown>` since technically people can just add objects as well.